dymos/examples/water_rocket/test/test_water_rocket.py

```diff
@@ -52,11 +51,12 @@ def test_water_rocket_height_for_docs(self):
 
         exp_out = traj.simulate(times_per_seg=10)
 
-        if not env_truthy('TESTFLO_RUNNING'):
-            plot_propelled_ascent(p, exp_out)
-            plot_trajectory(p, exp_out)
-            plot_states(p, exp_out)
-            plt.show()
+        # NOTE: only the last figure is shown in the generated docs
+        plot_propelled_ascent(p, exp_out)
+        plot_trajectory(p, exp_out)
+        plot_states(p, exp_out)
+
+        plt.show()
```

Comment:
did u intentionally remove the conditional to stop the plot from popping up during testing?